OTDB_Protocol.md

```diff
@@ -0,0 +1,225 @@
+
+**Methods of follow up**
+
+Participants will be followed up at the emergency department or on hospital discharge and at 24 hours, 30 days, 6 months, and 12 months. If a participant is still admitted he or she will be followed up in person, otherwise there will be telephone follow ups.
+
+#### Variables
+
+-	All-cause mortality at discharge, 24 hours, 30 days, 6 months and 12 months after arrival to participating hospital
+-	Date and time of death, up until 12 months after arrival to participating hospitals
+-	Health related quality of life
+-	Return to work
+-	Demographic variables
+-	Mechanism of injury
+-	Vital signs
+-	Comorbidities
+-	Investigations
+-	Injury descriptions
+-	Burns specific data like TBSA with depth of burns, inhalational burns, household burns etc
+
+#### Data sources
+
+Vital signs will be recorded by the project officers using hand held equipment when participants arrive to the emergency department. Demographics and follow up will be collected from participants, participant parties, and hospital records. Procedure data will be collected from hospital health care providers and hospital records.
+
+#### Ethical considerations
+
+This data bank will be implemented and maintained following the four core principles of medical ethics, i.e. respect for autonomy, non-maleficence, beneficence, and justice. 
+
+**Respect for autonomy**
+
+Patients included in the trauma data bank will receive a letter detailing the purpose and procedures. They are informed that participation is voluntary, and that they may withdraw at any time regardless of reason.
+
+**Non-maleficence**
+
+Patient data leakage and patient identification are the most significant risks to the patient population. The data collected will be stored in a locked cabinet to which only relevant personal has access. When data is transferred, all secondary storage devices (e.g. hard drives) will be password protected and encrypted. These actions will be undertaken to minimize these risks to the patient population.
+
+**Beneficence**
+
+The intention of this data bank is to provide data for future research and quality improvement in the field of trauma and trauma care. This research will in turn hopefully improve trauma outcomes by improving trauma patient care and trauma patient management.
+
+**Justice**
+
+All patients included in this study were treated regardless of age, sex, or prior medical history. Patients received the same level of care whether they choose to participate in the data bank or not.
+
+**Data management**
```

Comment:
This should not be a subheading under ethical considerations but separate.

Reply:
Noted and made the necessary changes